chart/compass/values.yaml

```diff
@@ -139,7 +139,7 @@ global:
       name: compass-pairing-adapter
     director:
       dir: dev/incubator/
-      version: "PR-3227"
+      version: "PR-3208"
```

Comment:
Update all components (with the latest director version from the PR) that depend on the director as well.